[StyleCop] Fix new warnings in Test TestHelpers

Aliandi opened the pull request:
- Add files for classes: RecognizerExtensions, TestModelExtensions, TestResources, TestResourcesExtensions, TestUtils
- Add ruleset to the project
- Document enums
- Add trailing comma
- Add and remove spacing when needed

ceciliaavila reviewed (changes requested):
Please check missing file testHelpers.cs

Aliandi committed:
[StyleCop] Fix new warnings in Test TestHelpers
- Add files for classes: RecognizerExtensions, TestModelExtensions, TestResources, TestResourcesExtensions, TestUtils
- Add ruleset to the project
- Document enums
- Add trailing comma
- Add and remove spacing when needed

ceciliaavila reviewed (approved):
LGTM

ceciliaavila committed:
Fixed spacing in RecognizerExtensions file

ceciliaavila committed:
Fixed remaining warnings in files.

- Replaced "" for string.Empty in TestUtils

ceciliaavila committed:
Merge remote-tracking branch 'origin/master' into fix/stylecop/dataDrivenTestTestHelpers

# Conflicts:
#	.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestContextExtensions.cs

ceciliaavila committed:
Merge remote-tracking branch 'origin/master' into fix/stylecop/dataDrivenTestTestHelpers

# Conflicts:
#	.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestContextExtensions.cs

ceciliaavila committed:
Merge remote-tracking branch 'origin/master' into fix/stylecop/dataDrivenTestTestHelpers
